daemon/mgr/container_exec.go

```diff
@@ -27,11 +27,18 @@ func (mgr *ContainerManager) CreateExec(ctx context.Context, name string, config
 		return "", fmt.Errorf("container %s is not running", c.ID)
 	}
 
+	envs, err := mergeEnvSlice(c.Config.Env, config.Env)
```

Comment:
first argu of mergeEnvSlice should `config.Env`

ping @knightXun cc @HusterWan

Reply:
I will correct it tonight